.github/workflows/release.yml

```diff
@@ -170,3 +170,19 @@ jobs:
                 data: fs.readFileSync(file)
               });
             }
+
+  Create-Issue-On-Fail:
+    permissions:
+      issues: write
+    runs-on: ubuntu-latest
+    needs: Publish-Release
+    if: failure() || cancelled()
```

Comment:
```suggestion
    if: (failure() || cancelled()) && github.event_name != 'workflow_dispatch'
```

Reply:
Really? I mean, workflow dispatch is still an issue right?

Reply:
If someone manually triggered the release workflow to test something, and it failed, I don't think an issue should be opened in the repository for everyone to see.

Reply:
Committed suggestion, per 👍 reaction above.